scripts/validate.mjs

```diff
@@ -2,40 +2,76 @@ import path from 'path'
 import Ajv from 'ajv'
```

Comment:
**chore:**
not sure if it exists in `.mjs` files but, it'd be nice to have some unit tests 😇

Reply:
Good suggestion. I added integration tests (i.e. running the command and checking its output) instead of unit tests as it will be simpler to cover bigger scope than unit tests. Let me know what you think!

Reply:
cool! 🚀